src/main/java/kr/codesqaud/cafe/domain/article/repository/ArticleRepository.java

```diff
@@ -1,14 +1,17 @@
 package kr.codesqaud.cafe.domain.article.repository;
 
 import kr.codesqaud.cafe.domain.article.Article;
+import kr.codesqaud.cafe.dto.Paging;
 
 import java.util.List;
 import java.util.Optional;
 
 public interface ArticleRepository {
     void save(Article article);
 
-    List<Article> findAll();
+    List<Article> findAll(Paging paging);
+
+    int allCount();
```

Comment:
```suggestion
    int count();
```

`nit`

org.springframework.data.repository.query

```java
	/**
	 * Returns the number of instances matching the given {@link Example}.
	 *
	 * @param example the {@link Example} to count instances for. Must not be {@literal null}.
	 * @return the number of instances matching the {@link Example}.
	 */
	<S extends T> long count(Example<S> example);
```

Reply:
수정했습니다.